examples/skr/TROUBLESHOOTING.md

```diff
@@ -14,6 +15,23 @@ Generate a new bearer token and copy it into the importkeyconfig.json.
 az account get-access-token --resource https://managedhsm.azure.net
 ```
 
+2. 
```

Comment:
Same comment above about numbering. Just remove the numbers and make these errors their own section.